aries-site/src/pages/learn/index.js

```diff
@@ -16,12 +25,73 @@ const Learn = () => (
       description={pageDetails.seoDescription}
       canonicalUrl="https://design-system.hpe.design/learn"
     />
-    <PageContent gap="large">
+    <PageContent>
       <Box pad={{ vertical: 'medium' }} justify="center" width="large">
         <Heading margin="none">{title}</Heading>
         <Paragraph size="large">{pageDetails.description}</Paragraph>
       </Box>
-      <CardGrid cards={cards} pad={{ bottom: 'large' }} headingLevel={2} />
+      <Data data={cards} pad={{ bottom: 'large' }}>
+        <DataSearch width={{ max: 'medium', width: '100%' }} />
+        <DataSummary />
+        <DataContext.Consumer>
+          {({ data }) => {
+            const gettingStarted = data.filter(
+              datum => datum.type === 'Getting started',
+            );
+            const howTo = data.filter(datum => datum.type === 'How-to guides');
+            const explanations = data.filter(
+              datum => datum.type === 'Explanations',
+            );
+            const references = data.filter(
+              datum => datum.type === 'References',
+            );
+
+            const results = [
+              {
+                heading: 'Getting started',
+                data: gettingStarted,
+              },
+              {
+                heading: 'How-to guides',
+                data: howTo,
+              },
+              {
+                heading: 'Explanations',
+                data: explanations,
+              },
+              {
+                heading: 'References',
+                data: references,
+              },
+            ];
+
+            return (
+              <Box gap="large" pad={{ top: 'medium' }}>
+                {results.map((type, index) =>
+                  type.data?.length ? (
+                    <Box gap="medium" key={index}>
+                      <Heading level={2} margin="none">
+                        {type.heading}
+                      </Heading>
+                      <Grid columns="medium" gap="medium">
```

Comment:
Is [Cards](https://v2.grommet.io/cards) an option here? I haven't played with it much, but curious what your experience with it might be.

Reply:
Cards has the DataContext built into it, but since we wanted to split up the results across a few different grids and have a heading for each section, managing it ourselves makes more sense in this case.